person.rb

```diff
@@ -0,0 +1,14 @@
+class Person
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options = {})
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+  end
+
+  def offer_high_five
```

Comment:
Nice work abstracting out the "HighFive" functionality out of the individual classes. Another option would be to place this methods in a module. That approach would allow you to include "HighFive" functionality in objects that don't inherit from Person.